Code/Editor/Controls/ConsoleSCB.cpp

```diff
@@ -1113,23 +1113,93 @@ void ConsoleVariableModel::ClearModifiedRows()
+                {
+                    cvar->Set(stringified.c_str());
+                }
+            }
+        }
+    }
+    return succeeded;
+}
+
 AZ::ConsoleCommandInvokedEvent::Handler ConsoleVariableEditor::m_commandInvokedHandler(
     [](AZStd::string_view command,
        const AZ::ConsoleCommandContainer&,
        AZ::ConsoleFunctorFlags,
        AZ::ConsoleInvokedFrom)
     {
-        if (command == AzToolsFramework::DocumentPropertyEditor::GetEnableDPECVarName())
+        if (command == AzToolsFramework::DocumentPropertyEditor::GetEnableCVarEditorName())
         {
             // the cvar editor pref changed, unregister the old and register the new
             AzToolsFramework::UnregisterViewPane(LyViewPane::ConsoleVariables);
             ConsoleVariableEditor::RegisterViewClass();
         }
-        else
+
+        // find the cvar that changed and keep the console informed
+        auto changedCVar = GetIEditor()->GetSystem()->GetIConsole()->GetCVar(AZStd::string(command).c_str());
+        if (changedCVar)
         {
-            // find the cvar that changed and keep the ConsoleVariableEditor informed
-            auto changedCVar = GetIEditor()->GetSystem()->GetIConsole()->GetCVar(AZStd::string(command).c_str());
-            if (changedCVar)
+            auto console = AZ::Interface<AZ::IConsole>::Get();
+            auto azConsoleCommand = console->FindCommand(command);
+            if (azConsoleCommand)
+            {
+                const bool handled =
+                    (SetCVarFromConsoleCommand<AZStd::string, CVAR_STRING>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::CVarFixedString, CVAR_STRING>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s8, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s16, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s32, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::s64, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::u8, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::u16, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::u32, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<AZ::u64, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<long, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<unsigned long, CVAR_INT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<float, CVAR_FLOAT>(changedCVar, azConsoleCommand) ||
+                     SetCVarFromConsoleCommand<double, CVAR_FLOAT>(changedCVar, azConsoleCommand)
+                     );
```

Comment:
This is missing the `bool` to `CVAR_INT` case.

Reply:
good catch, thanks! Fixed now